app/src/main/res/layout/fragment_ai_prompt.xml

```diff
@@ -102,45 +106,6 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content">
 
-                <com.google.android.material.card.MaterialCardView
-                    android:id="@+id/generateWithButton"
-                    style="@style/CardViewMail"
-                    android:layout_width="wrap_content"
-                    android:layout_height="0dp"
-                    android:layout_marginVertical="@dimen/marginStandardVerySmall"
-                    app:cardBackgroundColor="@color/backgroundColorSecondary"
-                    app:cardCornerRadius="@dimen/textButtonCornerRadius"
-                    app:layout_constrainedWidth="true"
-                    app:layout_constraintBottom_toBottomOf="parent"
-                    app:layout_constraintEnd_toStartOf="@id/generateButton"
-                    app:layout_constraintHorizontal_chainStyle="spread_inside"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toTopOf="parent">
-
-                    <LinearLayout
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:layout_gravity="center_vertical"
-                        android:gravity="center_vertical"
-                        android:orientation="horizontal"
-                        android:paddingHorizontal="@dimen/alternativeMargin">
-
-                        <TextView
-                            android:layout_width="0dp"
-                            android:layout_height="wrap_content"
-                            android:layout_weight="1"
-                            android:text="@string/aiGenerationTitlePrompt" />
```

Comment:
You still need the aiGenerationTitlePrompt string? Also take care of loco as well if you need to remove it.

And take this opportunity to check for other unused resources you've removed, because I've already seen a lot of them in just these few lines